frontend/ui/data/css_tweaks.lua

```diff
@@ -303,12 +303,49 @@ h1, h2, h3, h4, h5, h6 { hyphens: none !important; }
                 separator = true,
             },
             {
-                id = "ruby_inline";
-                title = _("Render <ruby> content inline"),
-                description = _("Disable handling of <ruby> tags and render them inline."),
-                css = [[
-ruby { display: inline !important; }
-                ]],
+                title = _("Ruby"),
+                {
+                    title = _("About ruby"),
+                    info_text = _([[
+Ruby characters are small glosses on the Asian languages to show the logographs' (Hanzi, Kanji, Hanja, etc) pronunciation.
+These tweaks allow small stylistic changes and larger ones to make ruby easier to read or ignore.]]),
+                    separator = true,
+                },
+                {
+                    id = "ruby_font_sans_serif";
+                    title = _("Nicer ruby font"),
```

Comment:
Is that nicer? Or just Sans-serif? :-P

Reply:
Well, don't want to add "sans-serif" words in the title to keep it not too technical and easier for translators/readers.
"Nicer" might be a question of taste, would "Cleaner" or "Smoother" be better ?
(It's just about having smooth slugs instead of ragged small ants above the bigger ant-ographs :)

Reply:
Smoother has the problem that it might make people think of anti-aliasing. Clean (as in without decorations) might work better.

But I think sans (serif) is fairly widely understood, and I'm not sure if you're right that it's too technical (cf. the popular character Sans from Undertale, and Comic Sans MS), while font smoothing is very much jargon.

Reply:
Well, I don't mind much.
I just considered it as a bag of minor stylistic update to the font (sans-serif being one update, regular weight another one, we might add some others :)
Also, the user may already be using our default NotoSansCJK, so already having ruby sans-serif. Having Nicer/Cleaner make it more abstract, he might just ends up thinking "well, I don't see the difference" and forget about it :)

Can you give me a full wording for the title of this item ?
@davidvelosa proposed `Stylize ruby w/ sans-serif font`, and I just didn't like "Stylize" and "w/", so had to find something else.

Reply:
Please, give me the full title, with uppercase first letter :) Should I put a verb before like "Use"?
It should look ok with the other ones in my screenshot :)

Reply:
No verb, I don't think that's necessary.

I would simply go with:

* Sans-serif font for ruby